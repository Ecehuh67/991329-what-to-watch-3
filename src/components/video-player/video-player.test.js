import VideoPlayer from './video-player';

const film = {
  title: `The corn`,
  image: `img/the corn.jpg`,
  preview: `https://player.vimeo.com/external/352010586.sd.mp4?s=2e414f16dc7380d1d48fd68c61db096db8ca1b06&profile_id=139&oauth2_token_id=57447761`
};

it(`VideoPlayer is rendered correctly`, () => {
  const tree = renderer.create(
      <VideoPlayer
        preview={film.preview}
        image={film.image}
        title={film.title}
        isActive={false}
      />,
      {
        createNodeMock: () => {
          return {};
        }
      }).toJSON();

  expect(tree).toMatchSnapshot();
});
