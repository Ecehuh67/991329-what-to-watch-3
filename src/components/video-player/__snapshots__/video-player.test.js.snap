// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer is rendered correctly 1`] = `
<video
  autoPlay={true}
  className="video-title"
  height="185"
  muted={true}
  src="https://player.vimeo.com/external/368010126.sd.mp4?s=ed2b36a42d2ac7b22000ed0f4ea187df7091423b&profile_id=139&oauth2"
  width="280"
/>
`;
